src/pages/Home/subcomponents/NewUserCity.js

```diff
@@ -1132,4 +1132,4 @@ NewUserCity.propTypes = {
   handleMapTypeChange: PropTypes.func
 };
 
-export default React.memo(NewUserCity);
+export default React.memo(NewUserCity);
```

Comment:
@GreenWithMV another minor thing, but better to end with newlines (if only to prevent this on Github).